awsfindingsmanagerlib/awsfindingsmanagerlib.py

```diff
@@ -311,11 +311,15 @@ def is_matching_resource_ids(self, resource_id_patterns) -> bool:
 
         Returns:
             True if any match is found, False otherwise.
+            An empty resource_id_patterns list will always return True, like Security Hub does per resource.
```

Comment:
True if any resource ID matches any pattern, or if patterns list is empty. False otherwise.